runtimes/eden/src/pallets_system.rs

```diff
@@ -102,10 +103,11 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = frame_system::Pallet<Runtime>;
 	type WeightInfo = crate::weights::pallet_balances::WeightInfo<Runtime>;
-	type MaxHolds = ConstU32<0>;
+	type MaxHolds = ConstU32<2>;
```

Comment:
What is the consequence of this change?

Reply:
There are two reasons so it must not be smaller than 2
```
test __construct_runtime_integrity_test::runtime_integrity_tests ... FAILED

failures:

---- __construct_runtime_integrity_test::runtime_integrity_tests stdout ----
thread '__construct_runtime_integrity_test::runtime_integrity_tests' panicked at /home/simson/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pallet-balances-28.0.0/src/lib.rs:552:13:
MaxHolds should be greater than or equal to the number of hold reasons: 0 < 2
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```